scadalts-ui/src/views/Alarms/EventList.vue

```diff
@@ -591,10 +600,13 @@ export default {
 			return value.charAt(0).toUpperCase() + value.slice(1)
 		},
 		truncate(input) {
-			if (input.length > 32) {
-				return input.substring(0, 32) + '...';
+			if (input.length > 45) {
+				return input.substring(0, 45) + '...';
 			}
 			return input;
+		},
+		clearHtml(str) {
+			return str.replace(/<[^>]*>?/gm, '').replaceAll('&nbsp;', ' ')
 		}
```

Comment:
Based on the [Vue documentation](https://vuejs.org/v2/guide/computed.html) we should keep those methods inside `computed` property of Vue object. But your solution also works